pkgs/top-level/perl-packages.nix

```diff
@@ -9867,11 +9867,11 @@ let
   # TODO: use CPAN version
```

Comment:
You can drop this comment

Reply:
I think this still applies? It's currently fetched from the exiftool webpage, but could be fetched from CPAN like many other Perl packages.

Reply:
Ah, I thought this was about version number, not URL where this is fetched from.

Reply:
The checksum matches what's on `mirror://cpan/authors/id/E/EX/EXIFTOOL/Image-ExifTool-12.00.tar.gz` so don't see a reason why it shouldn't use cpan.